src/components/Checkbox/Checkbox.tsx

```diff
@@ -4,7 +4,8 @@ import {withStylable} from 'wix-ui-core';
 
 interface CheckboxProps {
   skin?: 'standard' | 'error' | 'success';
-  size?: 'large' | 'small' | 'x-small';
+  size?: 'large' | 'small';
```

Comment:
no medium ?

Reply:
my mistake, it's supposed to be medium and large.